cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/SystemArguments.java

```diff
@@ -92,12 +91,12 @@ public static void setTransactionTimeout(Map<String, String> args, int timeout)
   }
 
   /**
-   * Returns the transction timeout based on the given arguments or, as fallback, the CConfiguration.
+   * Returns the transaction timeout based on the given arguments or, as fallback, the CConfiguration.
    *
-   * @returns the integer value of the argument system.data.tx.timeout, or if that is not given in the arguments,
-   *          the value for data.tx.timeout from the CConfiguration.
+   * @return the integer value of the argument system.data.tx.timeout, or if that is not given in the arguments,
+   *         the value for data.tx.timeout from the CConfiguration.
    * @throws IllegalArgumentException if the transaction timeout exceeds the transaction timeout limit given in the
-   *         CConfiguratuion.
+   *         {@link CConfiguration}.
```

Comment:
to make it clear, that is data.tx.max.timeout?

Reply:
Seems like so.